doc/nrf/known_issues.rst

```diff
@@ -0,0 +1,572 @@
+
+.. rst-class:: v130
+
+KRKNWK-6358: CoAP client sample provisioning issues
+  It is not possible to provision the :ref:`coap_client_sample` sample to servers that it cannot directly communicate with.
+  This is because Link Local Address is used for communication.
+
+.. rst-class:: v130
+
+KRKNWK-6408: ``diag`` command not supported
+  The ``diag`` command is not yet supported by Thread in the |NCS|.
+
+Zigbee
+======
+
+.. rst-class:: v130
+
+Potential delay during FOTA
+  There might be a noticeable delay (~220 ms) between calling the ZBOSS API and on-the-air activity.
+
+.. rst-class:: v130
+
+ZBOSS alarms inaccurate
+  On average, ZBOSS alarms last longer by 6.4 percent than Zephyr alarms.
+
+  **Workaround:** Use Zephyr alarms.
+
+Subsystems
+**********
+
+|BLE|
+=====
+
+SoftDevice Link Layer
+---------------------
+
+.. rst-class:: v110
+
+:option:`CONFIG_BT_HCI_TX_STACK_SIZE` requires specific value
+  :option:`CONFIG_BT_HCI_TX_STACK_SIZE` must be set to 1536 when selecting :option:`CONFIG_BT_LL_SOFTDEVICE`.
+
+.. rst-class:: v110
+
+:option:`CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE` requires specific value
+  :option:`CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE` must be set to 2048 when selecting :option:`CONFIG_BT_LL_SOFTDEVICE` on :ref:`central_uart` and :ref:`central_bas`.
+
+.. rst-class:: v110
+
+:option:`CONFIG_NFCT_IRQ_PRIORITY` requires specific value
+  :option:`CONFIG_NFCT_IRQ_PRIORITY` must be set to 5 or less when selecting :option:`CONFIG_BT_LL_SOFTDEVICE` on :ref:`peripheral_hids_keyboard`.
+
+.. rst-class:: v110
+
+Wait time required after a directed high duty cycle advertiser times out
+  When selecting :option:`CONFIG_BT_LL_SOFTDEVICE`, if a directed high duty cycle advertiser times out, the application might have to wait a short time before starting a new connectable advertiser.
+  Otherwise, starting the advertiser will fail.
+
+.. rst-class:: v110
+
+Assert risk after performing a DLE procedure
+  The :ref:`nrfxlib:softdevice_controller` 0.3.0-3.prealpha might assert when receiving a packet with an CRC error on LE Coded PHY after performing a DLE procedure where RX Octets is changed to a value above 140.
+
+Other issues
+------------
+
+.. rst-class:: v130
+
+NCSDK-5711: High-throughput transmission can deadlock the receive thread
+  High-throughput transmission can deadlock the receive thread if the connection is suddenly disconnected.
+
+.. rst-class:: v130
+
+NCSDK-5711: High-throughput transmission can deadlock the transmit thread
+    Bi-directional high-throughput traffic can deadlock the transmit thread.
+
+.. rst-class:: v120
+
+Only secure applications can use |BLE|
+  Bluetooth LE cannot be used in a non-secure application, for example, an application built for the ``nrf5340_dk_nrf5340_cpuappns`` board.
+
+  **Workaround:** Use the ``nrf5340_dk_nrf5340_cpuapp`` board instead.
+
+.. rst-class:: v120
+
+Peripheral HIDS keyboard sample cannot be used with SoftDevice Controller
```

Comment:
```suggestion
Peripheral HIDS keyboard sample cannot be used with nRF Bluetooth LE Controller
```
This one's tricky, because it's called SoftDevice Controller now (and thus links say SD Controller), but it was called nRF Bluetooth LE Controller at time of the release.